http-data/build.gradle

```diff
@@ -0,0 +1,5 @@
+description = 'Contains common data models exchanged between client/server'
+
+dependencies {
+
+}
```

Comment:
Does Gradle require this?  Can it just be removed?

Reply:
Seems not required, removed the unnecessary boiler plate in: 03cdaaa